src/extensions/renderer-api/index.ts

```diff
@@ -4,9 +4,11 @@
 import * as Component from "./components"
 import * as K8sApi from "./k8s-api"
 import * as Navigation from "./navigation"
+import * as KubeObjectStatus from "./kube-object-status"
 
 export {
   Component,
   K8sApi,
   Navigation,
+  KubeObjectStatus
```

Comment:
Do we really need separate namespace for this? 🤔

Reply:
I would rather not to add new namespace, but it does not fit very well in to any existing namespaces. But I'm open for suggestions.

Reply:
Everything else related to "KubeObject" is under `K8sApi`, would that make sense here?

Reply:
Yes. Moved it under `K8sApi`.